Automattic/wp-calypso
# A4A > Referrals: Implement agency client subscriptions list view

Closes https://github.com/Automattic/jetpack-genesis/issues/368
Closes https://github.com/Automattic/jetpack-genesis/issues/376

## Proposed Changes

This PR implements an agency client subscriptions list view.

NOTE: 

- [Design](https://www.figma.com/design/fuufP6VNfZXYmvLsTRWRlG/Referrals?node-id=6711-71510&m=dev)
- The `Cancel the subscription` functionality will be implemented in a different PR.


## Testing Instructions

1. Checkout to the branch locally and start the server.
2. Create a new WPCOM account, but don't sign up for A4A.
3. Get the ID(WPCOM ID) from this API that gets called when you log in: 
https://public-api.wordpress.com/rest/v1.1/me and note it down.
4. Login to your agency account > Go to Marketplace > Toggle Refer products > Select product > Go to Checkout > Enter the email ID of the newly created account > Open the network tab > Click the `Request payment from client` button > Get the ID of the API response that gets called when the button is clicked.
5. Follow the instructions here: 34bbb-pb
6. Logout and log in to your client account 
7. Go to /client/subscriptions
8. Verify that you see the Subscriptions list view as shown below. 

<img width="1353" alt="Screenshot 2024-06-11 at 3 28 47 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/57f9fb86-287a-4c54-99cf-c2d70383e6ff">

## Pre-merge Checklist
